swagger.json

```diff
@@ -40,9 +40,160 @@
         {
             "name": "Default",
             "description": "All Samsara API endpoints"
+        },
+        {
+            "name": "Users",
+            "description": "Access to user data"
         }
     ],
     "paths": {
+        "/users": {
+            "get": {
+                "tags": [
+                    "Default", "Users"
+                ],
+                "summary": "/users",
+                "description": "Get all users in your organization.",
+                "operationId": "get_users",
```

Comment:
Nit: would we call this `list_users` for consistency with `list collection` vs `get resource`?

Reply:
Sure!